docs/man-openmpi/man3/MPI_Info_create_env.3.rst

```diff
@@ -0,0 +1,80 @@
+.. _mpi_info_create_env:
+
+
+MPI_Info_create_env
+===================
+
+.. include_body
+
+:ref:`MPI_Info_create_env` - Creates a new info object with the same construction as MPI_INFO_ENV as created during :ref:`MPI_Init`  or :ref:`MPI_Init_thread` when the same arguments
+are used.
+
+
+SYNTAX
+------
+
+
+C Syntax
+^^^^^^^^
+
+.. code-block:: c
+
+   #include <mpi.h>
+
+   int MPI_Info_create_env(int argc, char *argv[], MPI_Info *info)
+
+
+Fortran Syntax
+^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE MPI
+   ! or the older form: INCLUDE 'mpif.h'
+   MPI_INFO_CREATE_ENV(INFO, IERROR)
+   	INTEGER	INFO, IERROR
+
+
+Fortran 2008 Syntax
+^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: fortran
+
+   USE mpi_f08
+   MPI_Info_create_env(info, ierror)
+   	TYPE(MPI_Info), INTENT(OUT) :: info
+   	INTEGER, OPTIONAL, INTENT(OUT) :: ierror
+
+
+OUTPUT PARAMETERS
+-----------------
+* ``info``: Info object created (handle).
+* ``IERROR``: Fortran only: Error status (integer).
+
+DESCRIPTION
+-----------
+
+:ref:`MPI_Info_create_env` creates a new info object and populates it with the same key/value pairs as are present in MPI_INFO_ENV.
+
+Note
+----
+
+:ref:`MPI_Info_create_env` is one of the few functions that can be called
+before :ref:`MPI_Init` and after :ref:`MPI_Finalize`.
+
+ERRORS
+------
+
+Almost all MPI routines return an error value; C routines as the value
+of the function and Fortran routines in the last argument.
+
+Before the error value is returned, the current MPI error handler is
+called. By default, this error handler aborts the MPI job, except for
+I/O function errors. The error handler may be changed with
+:ref:`MPI_Comm_set_errhandler`; the predefined error handler MPI_ERRORS_RETURN
```

Comment:
```suggestion
:ref:`MPI_Comm_set_errhandler`; the predefined error handler :ref:`MPI_ERRORS_RETURN`
```

Reply:
I don't think we actually have a page on `MPI_ERRORS_RETURN` (like we do for `MPI_INFO_ENV`).

Reply:
last change in this PR for a man page.  we need an overall solution to #10783 rather than just put in a quick fix here.

Reply:
Ok, fair enough.  Let's move ahead in this PR without the errhandling update, and we'll work that separately in #10783 for all man pages (including this one).